scripts/start.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-console */
+import yargs from 'yargs'
+import qrcodeTerminal from 'qrcode-terminal'
+import internalIp from 'internal-ip'
+import * as $ from 'shelljs'
+
+$.config.fatal = true
+
+const DEFAULT_PORT = 18000
+
+const argv = yargs(process.argv.slice(2))
+  .usage('Usage: $0 [--watch]')
+  .options({
+    watch: {
+      alias: 'w',
+      describe: 'Watch for changes, defaults to true',
+      type: 'boolean',
+      default: true,
+    },
+    port: {
+      alias: 'p',
+      describe: `Port to listen on, defaults to ${DEFAULT_PORT}`,
+      type: 'number',
+      default: DEFAULT_PORT,
+    },
+  })
+  .parseSync()
+
+const colors = {
+  bold: [1, 22],
+  italic: [3, 23],
+  underline: [4, 24],
+  inverse: [7, 27],
+  white: [37, 39],
+  grey: [90, 39],
+  black: [30, 39],
+  blue: [34, 39],
+  cyan: [36, 39],
+  green: [32, 39],
+  magenta: [35, 39],
+  red: [31, 39],
+  yellow: [33, 39],
+}
+
+/**
+ * Add colors to the specified string, for colorized output in terminals
+ */
+function stylize(str: string, color: keyof typeof colors) {
+  if (!str) {
+    return ''
+  }
+
+  const codes = colors[color]
+  if (codes) {
+    return '\x1B[' + codes[0] + 'm' + str + '\x1B[' + codes[1] + 'm'
+  }
+  return str
+}
+
+void (async () => {
```

Comment:
the `main().catch(err => { whatever }).then()` pattern we usually follow seems cleaner because it doesn't leave unresolved promises.  Is there a reason to not use that?

Reply:
Ah, I don't actually need top level async/await so I removed it.